modules/nw-anp-audit-logging-concept.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift_network_security/AdminNetworkPolicy/logging-anp-policy.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-anp-audit-logging_{context}"]
+= AdminNetworkPolicy audit logging
+
+The OVN-Kubernetes network plugin uses Open Virtual Network (OVN) access control list (ACL) to manage egress firewalls and network policies. Audit logging exposes `allow` and `deny` ACL events for `NetworkPolicy` and `BaselineAdminNetworkPolicy` custom resources (CR). Logging also exposes `allow`, `deny`, and `pass` ACL events for `AdminNetworkPolicy` (ANP) CR.
+
+You can configure the destination for audit logs, such as a syslog server or a UNIX domain socket.
+Regardless of any additional configuration, an audit log is always saved to `/var/log/ovn/acl-audit-log.log` on each OVN-Kubernetes pod in the cluster.
+
+Audit logging is enabled per `AdminNetworkPolicy` CR by annotating an ANP policy with the `k8s.ovn.org/acl-logging` key such as in the following example:
+
+.Example of annotation for `AdminNetworkPolicy` CR
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: AdminNetworkPolicy
+metadata:
+ annotations:
+  k8s.ovn.org/acl-logging: '{ "deny": "alert", "allow": "alert", "pass" : "warning" }'
+ name: anp-tenant-log
+spec:
+ priority: 5
+ subject:
+  namespaces:
+     matchLabels:
+        kubernetes.io/metadata.name: tenant
+ egress:
+ - name: "deny-ns-tenant-1"
+  action: "Deny"
+  to:
+  - namespaces:
+     matchLabels:
+        kubernetes.io/metadata.name: tenant_1
+ - name: "allow-ns-tenant-2"
+  action: "Allow"
+  to:
+  - namespaces:
+     matchLabels:
+        kubernetes.io/metadata.name: tenant_2
+ - name: "pass-ns-tenant-3"
+  action: "Pass"
+  to:
+  - namespaces:
+     matchLabels:
+        kubernetes.io/metadata.name: tenant_3
+----
+====
+
+Logs are generated whenever a specific ACL meets the criteria set in your annotation. In the example, an event in which the `tenant` namespace is denied egress to the `tenant_1` namespace a log is generated.
+
+The logging format is compatible with syslog as defined by RFC5424. The syslog facility is configurable and defaults to `local0`. An example log entry might resemble the following:
```

Comment:
Do you want to move this to where we describe policyAuditConfig:?

Reply:
I'm not sure I understand what you mean. I was trying to follow the structure of the [here](https://docs.openshift.com/container-platform/4.15/networking/ovn_kubernetes_network_provider/logging-network-policy.html).